features/zoek-fout-cases.feature

```diff
@@ -236,5 +229,5 @@ Rule: Om privacy en security redenen moet een bevraging van reisdocumenten worde
     | DELETE       |
     # | CONNECT      | een CONNECT aanroep wordt niet gebruikt om te bevragen
     # | HEAD         | een HEAD response bevat geen body
-    | OPTIONS      |
-    | TRACE        |
+    | OPTIONS |
```

Comment:
hiermee is uitlijning niet meer correct. Waarschijnlijk ontstaan doordat er comments tussen staan?